src/client/components/Title/TitleDesc.stories.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+import  TitleDesc  from './TitleDesc';
+
+
+export default { title: 'Title component' };
+
+export const title = () => <TitleDesc title="Here goes the title" description="Heres goes the description" />;
```

Comment:
In this case, it should be more like this:

```
import React from 'react';

import TitleDesc from './TitleDesc';

export default { title: 'Title component' };

export const title = () => (
  <TitleDesc
    title="Here goes the title"
    description="Heres goes the description"
  />
);

```

Reply:
Thank you valentina, i think you did a good review. in my concept when someone as reviewer suggest a possible solution this is the best way as a teamwork helping each other, instead of say this is weird, or something like that.